### ISSUE TITLE
Can't configure what members to skip

### ISSUE THREAD

**Issue by w3con** [mishkinf/vscode-goto-next-previous-member#9]
Hey there,
thanks for great extension.

Unfortunately I can't configure it:

settings.json:
```
...
"gotoNextPreviousMember.symbolKinds": ["function", "method"],
...
```
doesn't work. Empty array works fine.

**Comment by TheTastefulToastie** [comment]
What programming language? What OS?
If you try `"variable"` does that work?

**Comment by w3con** [comment]
MacOS, JS
thanks!

**Comment by w3con** [comment]
`  "gotoNextPreviousMember.symbolKinds": ["variable"],` is exactly the same as with empty array – it jumps from variable to variable.

**Comment by TheTastefulToastie** [comment]
Thanks for the info.

I tried to replicate this with a JS file on MacOS.

With `"gotoNextPreviousMember.symbolKinds": ["variable"]` in `settings.json` it jumps only to variables and not to other symbols like functions.

I also tried `"gotoNextPreviousMember.symbolKinds": ["function", "method"]` which worked for me as well.

I'm grasping at straws here but the extension version you've got is definitely 0.0.5 ?

Maybe also try a blank `settings.json` with only the `gotoNextPreviousMember.symbolKinds` line in it.
